docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+            - [ ] Identify common questions across programs (Ex. Name, Address, income).  
+            - [ ] Explore ways to consolidate these questions to streamline reuse.  
+            - [ ] Consider creating a “Common Intake” form as a “Get Started”.  
+      - [ ] For each program:  
+            - [ ] Decide on who the program leader/owner and primary point of contact will be.  
+            - [ ] Decide how applications to the program will be reviewed. Will they be reviewed in CiviForm, or pulled into an external system? If an external system, identify the technical PoCs.
+
+- [ ] **Join the CiviForm community**  
+      - [ ] Understand how the CiviForm community works.  
+            - [ ] [Community member responsibilities](https://docs.civiform.us/governance-and-management/governance/roles-committees-and-responsibilities)  
+            - [ ] [Governance Processes](https://docs.civiform.us/governance-and-management/governance/governance-processes)  
+      - [ ] Learn about  the [CiviForm community's communication channels](https://docs.civiform.us/governance-and-management/governance/communication).  
+            - [ ] Join [CiviForm Slack](https://civiform.slack.com/) and introduce yourself in the \#introductions channel.  
+            - [ ] Join the [mailing lists](https://docs.civiform.us/governance-and-management/governance/communication) as appropriate.  
+            - [ ] Create a [Github](https://github.com/) account so you can [file issues for bugs and/or feature requests](https://github.com/civiform/civiform/issues).  
+      - [ ] Join the CiviForm community meetings:  
+            - [ ] Every other week join the CiviForm Product Design Committee to learn about new features or ask questions of fellow governments using CiviForm.  
+            - [ ] Monthly join the CiviForm Governance group to learn about best practices, new features and tools, as well as how other civic entities are leveraging CiviForm.
+
+## Preparation for Launch 
+
+- [ ] Review the [form building how-to-guide](https://docs.civiform.us/user-manual/civiform-admin-guide/working-with-programs/create-a-program) to begin building your first program  
+      - [ ] User-centric design: Create forms that are easy to understand and complete, using [plain language](http://plainlanguage.gov).  
+      - [ ] Keep  accessibility standards (e.g., WCAG) in mind when creating your forms.  
+      - [ ] Consider which languages Program questions and answers will be translated to and which languages [CiviForm supports natively](https://docs.civiform.us/user-manual/civiform-admin-guide/manage-translations-for-programs-and-questions).  
+            - [ ] Find translation resources and add time for translations to your project timeline.  
+      - [ ] Data validation: Implement program checks (such as phone number formats and hints to users “select only two choices” to prevent errors and ensure data accuracy.  
```

Comment:
**nitpick:** No close parentheses

Reply:
Thanks. Can you make a comment in the Google Doc for now?